wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -13,7 +13,9 @@ end
 
 namespace :wagn do
   desc "create a wagn database from scratch"
-  task :create do
+  task :recreate do
```

Comment:
Point on task names.  We need something that means 'clear and reload the data, but don't drop and add the schema', vs. what create means originally.  In the embedded case, the equivalent of reloading schema will be to revert the migration that added cards, and then run a migration (possibly different) that add the current cards schema.  Maybe we need Card.embedded? or similar so code/config/tasks can know which it is?